packages/bits/src/lib/wizard-v2/public-api.ts

```diff
@@ -10,3 +13,7 @@ export * from "./wizard-horizontal/wizard-horizontal.component";
 export * from "./wizard-step/wizard-step.component";
 export * from "./wizard-step-header/wizard-step-header.component";
 export * from "./wizard-vertical/wizard-vertical.component";
+
+export interface IWizardState {
```

Comment:
Please create a dedicated types.ts file with this interface. public-api.ts should not directly contain any type / class definitions.